.docker/Dockerfile.php

```
@@ -5,7 +5,7 @@
 
 FROM amazeeio/php:${PHP_IMAGE_VERSION}-fpm-${LAGOON_IMAGE_VERSION}
 
-RUN apk add --no-cache --update clamav clamav-libunrar \
+RUN apk add --no-cache --update clamav clamav-libunrar --repository http://dl-3.alpinelinux.org/alpine/edge/main \
```

This should use the cdn version, not the direct link to the repo

```suggestion
RUN apk add --no-cache --update clamav clamav-libunrar --repository http://dl-cdn.alpinelinux.org/alpine/edge/main/ \
```